internal/app/server/server.go

```diff
@@ -59,6 +62,26 @@ func Run(ctx context.Context, network string, cfg *config.ServiceConfig) error {
 		}),
 	}
 
+	var tracer *trace.TracerProvider
+	if cfg.EnableTrace {
+		log.Printf("Enabling CloudTrace exporter with sample rate: %f\n", cfg.ServerConfig.TraceSampleRate)
+
+		grpcOptions = append(grpcOptions,
```

Comment:
Should we be checking for the grpc collector flag before appending these options?

Reply:
Those are not for the collector but for the server, we want them as long as tracing is enabled. Although if both enableGRPCCollector and enableHTTPCollector were disabled tracing would do anything useful - which is a shortcoming but I hesitated to remove pre-existing EnableTrace. Could use stdout print as a fallback or showing a warning if tracing was enabled without collectors?